chat.py

```diff
@@ -25,27 +26,33 @@ def send_messages_to_chat(projects_by_owner):
     for owner in projects_by_owner.keys():
         user_to_mention = USERS_MAP.get(owner, owner)
         message = "Hey *@{}*, I've noticed you are one of the owners of the following projects:\n\n".format(user_to_mention)
+        send_message_to_this_owner = False
         for project in projects_by_owner.get(owner):
             project_id = project.get('projectId')
             org = ORGS_NAME_MAPPING.get(project.get('parent').get('id'))
             created_days_ago = int(project.get('createdDaysAgo'))
             cost = project.get('costSincePreviousMonth', 0.0)
             currency = project.get('costCurrency', '$')
             emoji = ''
-            if cost > COST_ALERT_THRESHOLD:
-                emoji = ' ' + COST_ALERT_EMOJI
-            message += "- `{}/{}` created `{} days ago`, costing *`{}`* {}.{}\n".format(org, project_id, created_days_ago, cost, currency, emoji)
+            if cost <= COST_MIN_TO_NOTIFY:
+                logger.debug('- `{}/{}`, will not be in the message, due to its cost being lower than the minimum warning value'.format(owner, project_id))
+            else:
+                if cost > COST_ALERT_THRESHOLD:
+                    emoji = ' ' + COST_ALERT_EMOJI
+                    send_message_to_this_owner = True
```

Comment:
You should send the message if the cost is higher than COST_MIN_TO_NOTIFY.

Here lies a bug, because you are sending the message if:
1.  cost is higher than COST_MIN_TO_NOTIFY
2. *AND* cost is higher than COST_ALERT_THRESHOLD.

Reply:
these changes are made in commit 239fa41